### ISSUE TITLE
Should we show "headless" evidence?

### ISSUE THREAD

**Issue by larrybabb** [clingen-data-model/clingen-interpretation#65]
@bpow @cbizon 

Not sure if "headless" is a great term but a real eye-catcher.

In our Criterion examples we did capture the "strength" that the assessed criterion would have if and when it would be used in an interpretation.  But since we are not creating VarInterp parent structures for every Criterion Assessment example, I was not sure if I should create the "Evidence Line" that would be needed to connect these "headless" criterion assessments to their conceptual variant interpretations.

As such, there is no place to put the "strength" for the plethora of CritAssess examples, many of which demonstrate the "tweaking" of strength from the default for the corresponding ACMG rule.

So, question... Should I add in the Evidence Lines for these "headless" CritAssess examples so that we can capture these interesting "strength" data points, thus creating "headless" evidence lines?  And, would we show these parent evidence lines in some type of CritAssess example excerpts to show the portion of the VarInterp JSON that would contain the CritAssess content along with it's wrapping evidence line and strength value?

Let me know.  I will assume yes and begin doing this in the next few days if I don't hear otherwise.

**Comment by bpow** [comment]
Yes, this is the awkward thing about putting the strength on the
EvidenceLine that we talked about when first putting strength there. It
makes the EL into a "mini-assertion" (or maybe a "meta-assertion"), about
how relevant that EL is for the higher
​-order​
assertion. Which is why we needed to also allow for a Contribution on EL
since that could be assigned by either the person making the assertion, or
(in the "headless" case) by the agent making the CriterionAssessment.

The awkward part about a headless EL it's that an EL is a reified
connection between
​ an Assertion and its underlying Information. That definition loses
something when one of the nodes of the reified connection doesn't exist...

To try to express Chris's thoughts-- in the VCI, we don't think this will
be an issue because a CriterionAssessment will only be made in the context
of a VariantInterpretation, so nothing would be "headless". I do think
there is a potential use case for having a headless EL, though, especially
​for criteria that may be completely computationally assessable. And I
think it is also reasonable for documentation purposes to have ELs not
associated with VariantInterpretations as examples of how things might work.

TL;DR: given that the strength gets assigned at the EL level, I think we
have to allow "headless" EL, at least for documentation purposes.

**Comment by cbizon** [comment]
If we don't want a headless EL, then we can put a dummy variant interpretation above the EL.  The strength of the EL really should be a function of what the interpretation is.  For instance, maybe the same met criteria assessment will be stronger/weaker when the interpretation is phrased for slightly different mendelian conditions (bringing in or excluding phenotypes or broadening conditions).

**Comment by bpow** [comment]
I would be OK with @cbizon's suggestion, too. We could just make a VUS VariantInterpretation for a certain condition above CriterionAssessments that are not currently associated with a VI. It perhaps falsely inflates the number of "full VariantInterpretations" we have, though.

**Comment by larrybabb** [comment]
Hmmm... I'm not sure how we should deal with these "dummy" var interps to fill the gap we are dealing with on this topic.

I can do this, but then it means making up enough data for the remainder of the Var interp to have it seem reasonable (e.g. text blurb, etc..)  I think I can do some of it but there's going to be lots of dummy var interps to meet the long list of crit assessements.

And, if we do this then should we somehow tag these var interps so that we don't generate examples for them or at least be able to distinguish them from the "full" ones.  We will need a way to clearling indicate when an example is partial (i think).

**Comment by larrybabb** [comment]
We will use this discussion to enhance our documentation around use of evidence lines with variant interps and criterion assesssments that may have no evidence items below them.
